src/main/scala/performanceanalysis/util/Utils.scala

```diff
@@ -0,0 +1,16 @@
+package performanceanalysis.util
+
+/**
+  * Created by seeta on 5/23/16.
+  */
+object Utils {
+  /**
+    * This function returns an array of lines. It treats
+    * \r\n, \n, \r as a new line
+    * @param input to be split
+    * @return array of lines
+    */
+  def split(input: String):Array[String] = {
```

Comment:
I know it might be bit too much write separate function for single line. I would like to to test all possible cases so I extracted it out.